examples/webserver/streaming/README.md

```diff
@@ -7,13 +7,13 @@ to the size of the file being uploaded or downloaded.
 
 ## Build and run
 
-```shell
+```bash
```

Comment:
I would not change it back to `bash` because `shell` is more generic. The reason, why I changed it to `shell` is to unify the notation across the whole examples' README. IntelliJ Idea allows running the code notated with `shell` (however either with `bash`) directly from the IDE. But as soon as the `shell` notation means any shell, I think it's more suitable than `bash` which is the only implementation of a `shell`.

Reply:
Fixed.

This was due to a cherry-pick from 3.x (it looks like in dev-3.x we are still using bash).